src/test/kotlin/hr/foi/pop/backend/services/UserServiceTest.kt

```diff
@@ -90,4 +92,58 @@ class UserServiceTest {
         val userWithBadUsername = templateRequestBodyForTesting.copy(username = "bad")
         assertThrows<UserCheckException> { userService.registerUser(userWithBadUsername) }
     }
+
+    @Test
+    @Transactional
```

Comment:
You can annotate the whole class with this and then every method gets to be `Transactional`. 😃